cpp/Glacier2/session/InMemoryPokeStore.cpp

```diff
@@ -0,0 +1,18 @@
+
+#include "InMemoryPokeStore.h"
+
+using namespace Server;
+using namespace std;
+
+void
+InMemoryPokeStore::saveCollection(const string& userId, CatchThemAll::PokemonList pokemon)
+{
+    _store[userId] = std::move(pokemon);
```

Comment:
This implementation seems different than the other languages.
In the others passing an empty list causes us to remove the entry, whereas here we never remove entries.

I don't think this is 'wrong', just different from the others.